build/common/init_command.sh

```diff
@@ -37,6 +37,9 @@ elif [ "$MODULE_NAME" = 'nginx' ]; then
         envsubst_create_config /etc/nginx/openwisp.template.conf http
         nginx -g 'daemon off;'
     fi
+elif [ "$MODULE_NAME" = 'celery' ]; then
+    python services.py database redis dashboard
+    celery -A openwisp worker -l info
```

Comment:
> are we sure that this service does not expose URLs right?

This container is not exposed outside the cluster! :smile: 

> It should only run the celery worker, but I don't see where the celery worker is being launched.
Can you point it out please?

When  `"$MODULE_NAME" = 'celery'` -> `celery -A openwisp worker -l info`